### ISSUE TITLE
Standalone, exported types

### ISSUE THREAD

**Issue by EcksDy** [OpenAPI-Qraft/openapi-qraft#99]
Great work on the tool!
 
I'm exploring it as a replacement for `openapi-generator`, and noticed that I can't access the dto types that were generated from my routes directly.

Currently I use a workaround:
```ts
type CreateDto = components['schemas']['CreateSomethingDto'];
```

The use case I have, is using these types to annotate forms for example.

Am I missing a config, or is it not possible at the moment?

**Comment by radist2s** [comment]
Thank you for your interest.

Under the hood, Qraft uses the [`openapi-typescript`](https://openapi-ts.dev/cli) utility for generating types from OpenAPI specifications. This utility generates similar types, and consequently, Qraft is compatible with them.

If `openapi-typescript` adds support for generating exportable component types, Qraft will also support this feature.

**Comment by radist2s** [comment]
@EcksDy, I've just released [`@openapi-qraft/cli@1.14.0-beta.5`](https://github.com/OpenAPI-Qraft/openapi-qraft/releases/tag/%40openapi-qraft%2Fcli%401.14.0-beta.5). My respects for your contribution! I will release a stable release any day now.
